lib/aws_loader.rb

```diff
@@ -1,9 +1,9 @@
 require 'yaml'
-require_relative '../config/s3_store'
 module AwsLoader
   
-  AWS_PATH = File.join(QIt.root, 'config/aws.yml')
+  AWS_PATH = File.join(Dir.pwd, 'config/aws.yml')
```

Comment:
Use a similar style as `q_it` code. Have a `S3Store.root` or something to that effect.

Reply:
I used Store.root initially. Its going into this dependency loop, and throws an error while requiring the file and is not able to declare the constant. So changed it to `Dir.pwd`.

Reply:
Try `App.root` or a similar generic term for the application rather than have `QIt.root` or `Store.root`.